src/build/incrementality_test.go

```diff
@@ -51,6 +51,7 @@ var KnownFields = map[string]bool{
 	"TestOutputs":                 true,
 	"Stamp":                       true,
 	"OutputDirectories":           true,
+	"RuleMetadata":                true,
```

Comment:
should this be in one of the other sections of this file? looks like it doesn't actually contribute to the rule hash (which I think is OK since it all happens at parse ime)